### PR TITLE
draft for review

### PR DESCRIPTION
**Pull request by julielegendre**

### CODE REVIEW

**Comment by julielegendre** [comment]
@indaos could you review please?

**Comment by indaos** [comment]
> @indaos could you review please?

Can we postpone it to next week? there will be a little more information but I'm not finished yet.

**Comment by julielegendre** [comment]
> Can we postpone it to next week? there will be a little more information but I'm not finished yet.

No problem. Feel free to add directly to the PR.

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# ios-sdk-xcframeworks
\ No newline at end of file
+# Unblu iOS mobile SDK Swift Package
+
+The Unblu iOS mobile SDK serves as a way to integrate the Unblu platform into native iOS apps.
+
+Unblu delivers the Unblu iOS mobile SDK as a Swift package on GitHub: https://github.com/unblu/ios-sdk-xcframeworks
```
Comment (indaos) [inline README.md:5]: Swift packages

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+# Installing
+
+To add the Unblu iOS mobile SDK Swift package to your Xcode project:
+
+1. In Xcode, go to **File** > **Add Packages**.
+2. Enter the following repository URL: https://github.com/unblu/ios-sdk-xcframeworks
+3. In the **Dependency Rule** dropdown, select **Up To Next Major Version**.
+4. Select the project you want to add it to.
+5. Click **Add Package**.
+6. In the window that appears, choose the **UnbluMobileSDK** package product and click **Add package**.
```
Comment (indaos) [inline README.md:16]: There will be several packages here, they must select the necessary ones, then they will appear....

Comment (julielegendre) [reply README.md:16]: What are the necessary packages? Ideally, we should list them.

**Pull request merged**